lib/components/customs/CustomPicture.vue

```diff
@@ -154,6 +159,15 @@ function filterBySupportedMimeTypes (sources) {
 function isWebp ({ type }) {
   return type === getMimeTypeByFormat('webp');
 }
+
+function doPolyfill () {
+  if ('picturefill' in global) {
+    global.picturefill();
+  }
+  if ('objectFitImages' in global) {
+    global.objectFitImages();
```

Comment:
please pass the element to the polyfill function else the whole html document will be parsed